cmd/zetae2e/config/localnet.yml

```diff
@@ -54,6 +54,12 @@ policy_accounts:
     bech32_address: "zeta142ds9x7raljv2qz9euys93e64gjmgdfnc47dwq"
     evm_address: "0xAa9b029BC3EFe4c50045Cf0902c73aAa25b43533"
     private_key: "0595CB0CD9BF5264A85A603EC8E43C30ADBB5FD2D9E2EF84C374EA4A65BB616C"
+observer_relayer_accounts:
```

Comment:
Are these information necessary when running non Solana tests? We should avoid breaking E2E for live network tests

Reply:
not sure what you meant. Do you mean config serialization or something?